tests/test_monitor.py

```diff
@@ -100,3 +100,8 @@ def test_new_clients_appear_in_focused_monitor(hlwm):
     assert hlwm.get_attr('tags.by-name.tag2.client_count') == '1'
     assert hlwm.get_attr('tags.by-name.default.client_count') == '0'
     # TODO: Assert that client's winid is in tag2 (not yet possible)
+
+def test_detect_monitors_does_not_crash(hlwm):
```

Comment:
Seems reasonable to "test" `detect_monitors` this way, but there should be two blank lines above. ;-)